contracts/dnr/dnr.go

```diff
@@ -70,11 +70,11 @@ func NewEthereumDarkNodeRegistrar(context context.Context, clientDetails *connec
 		binding:                  contract,
 		tokenBinding:             renContract,
 		darkNodeRegistrarAddress: clientDetails.DNRAddress,
-	} , nil
+	}, nil
 }
 
 // Register registers a new dark node
-func (darkNodeRegistrar *EthereumDarkNodeRegistrar) Register(_darkNodeID []byte, _publicKey []byte) (*types.Transaction, error) {
+func (darkNodeRegistrar *EthereumDarkNodeRegistrar) Register(_darkNodeID []byte, _publicKey []byte, _bond *big.Int) (*types.Transaction, error) {
```

Comment:
I assume the `bindings` package, which is auto-generated, automatically labels the parameters using the leading underscore. In our own implementations though we need to be consistent with the Golang naming convention.

Reply:
completed.